docs/tutorials/openshift.md

```diff
@@ -2,10 +2,6 @@
 This tutorial describes how to configure ExternalDNS to use the OpenShift Route source.
 It is meant to supplement the other provider-specific setup tutorials.
 
-### Prepare ROUTER_CANONICAL_HOSTNAME in default/router deployment
-Read and go through [Finding the Host Name of the Router](https://docs.openshift.com/container-platform/3.11/install_config/router/default_haproxy_router.html#finding-router-hostname).
-If no ROUTER_CANONICAL_HOSTNAME is set, you must annotate each route with external-dns.alpha.kubernetes.io/target!
-
```

Comment:
This still needs to be done to support OCP 3.x (as a fallback).
For sure it needs to be stated that this is only needed for OCP 3.x and OCP 4.x is "fully automated".

Reply:
Agreed and resolved.